docs/resources/Application.md

```diff
@@ -4,32 +4,35 @@
+| id                                 | snowflake                                                                  | ID of the app                                                                                                         |
+| name                               | string                                                                     | Name of the app                                                                                                       |
+| icon                               | ?string                                                                    | [Icon hash](#DOCS_REFERENCE/image-formatting) of the app                                                              |
+| description                        | string                                                                     | Description of the app                                                                                                |
+| rpc_origins?                       | array of strings                                                           | List of RPC origin URLs, if RPC is enabled                                                                            |
+| bot_public                         | boolean                                                                    | when `false`, only the app owner can add the app to guilds                                                            |
+| bot_require_code_grant             | boolean                                                                    | when `true`, the app's bot will only join upon completion of the full OAuth2 code grant flow                          |
+| bot?                               | partial [user](#DOCS_RESOURCES_USER/user-object) object                    | Partial user object for the bot user associated with the app                                                          |
+| terms_of_service_url?              | string                                                                     | URL of the app's Terms of Service                                                                                     |
+| privacy_policy_url?                | string                                                                     | URL of the app's Privacy Policy                                                                                       |
+| owner?                             | partial [user](#DOCS_RESOURCES_USER/user-object) object                    | Partial user object for the owner of the app                                                                          |
+| summary *(deprecated)*             | string                                                                     | **deprecated and will be removed in v11.** An empty string.                                                           |
```

Comment:
```suggestion
| summary *(deprecated)*             | string                                                                     | **Deprecated and will be removed in v11.** An empty string.                                                           |
```

Reply:
If this is deprecated and always an empty string, maybe we can just undocument this field?